src/cards.ts

```diff
@@ -51,9 +51,25 @@ export class CardGenerator {
 		context2d.fillStyle = '#ffffff'
 
 		context2d.fillText(`Remote: ${remoteIp}`, 10, height - 10)
-		context2d.fillText(`Status: ${status}`, 10, height - 30)
+		context2d.fillText(`local: ${getIPAddress()}`, 10, height - 30)
+		context2d.fillText(`Status: ${status}`, 10, height - 50)
 
 		// return result
 		return Buffer.from(context2d.getImageData(0, 0, width, height).data)
 	}
 }
+
+import { networkInterfaces } from 'os'
+function getIPAddress() {
```

Comment:
In the future, I would like to make this be a bit reactive, so that the ip address updates as it changes.
That can be a later PR